docs/src/main/asciidoc/virtual-threads.adoc

```diff
@@ -137,10 +137,14 @@ It can be reasonably easy to avoid these situations in your code, but verifying
 Typically, while experimenting with virtual threads, we realized that old versions of the link:{pgsql-driver}[postgresql-JDBC driver]
 results in frequent pinning.
 Most JDBC drivers still pin the carrier thread.
-Even worse, lots of widespread libraries are pinning and would require code changes.
+Even worse, many libraries require code changes.
 
 For more information, see link:https://quarkus.io/blog/virtual-thread-1/[When Quarkus meets Virtual Threads]
 
+IMPORTANT: This information about pinning cases applies to PostgreSQL JDBC driver 42.5.4 and earlier.
```

Comment:
I think the position is misleading. I read it like: Oh! Cool, they fix all the pining.
It's not the case. I would just replace `old versions` with `versions preceding 42.6.0`

Reply:
Thanks, @cescoffier. Is this the change you were suggesting?

Reply:
> Typically, while experimenting with virtual threads, we realized that old versions of the link:{pgsql-driver}[postgresql-JDBC driver]

would become:

> Typically, while experimenting with virtual threads, we realized that versions older than 42.6.0 of the link:{pgsql-driver}[postgresql-JDBC driver]